stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/ThreadParticipants.kt

```diff
@@ -1,44 +1,79 @@
 package io.getstream.chat.android.compose.ui.components.messages
 
-import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
-import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.zIndex
 import coil.compose.rememberImagePainter
 import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.compose.state.messages.MessageAlignment
 import io.getstream.chat.android.compose.ui.components.avatar.Avatar
 import io.getstream.chat.android.compose.ui.theme.ChatTheme
 
+private const val DEFAULT_PARTICIPANTS_LIMIT = 4
```

Comment:
I think we should discuss where constants should go. In the Compose sources they are at the bottom of each file not to distract attention from the main Composable exposed in a file.

Reply:
Good point - we can keep it on the bottom! :]